photon-client/src/main/scala/com/linkedin/photon/ml/data/avro/ResponsePredictionFieldNames.scala

```diff
@@ -17,12 +17,7 @@ package com.linkedin.photon.ml.data.avro
 /**
  * Response prediction format fields names.
  */
-object ResponsePredictionFieldNames extends FieldNames {
-  val features: String = "features"
-  val name: String = "name"
-  val term: String = "term"
-  val value: String = "value"
-  val response: String = "response"
-  val offset: String = "offset"
-  val weight: String = "weight"
-}
+object ResponsePredictionFieldNames extends AvroFieldNames {
+
+  override val RESPONSE: String = "response" // do we really need a different name?
```

Comment:
"Not liking them" is not a reasonable, rational argument. I "like" inline comments very much: they are real convenient to avoid disturbing the flow of the code. For this comment, it's the todo that we should not have "label" and "response". But it's captured somewhere else, so I'm ok to remove it here. I completely "reserve the right" to use inline comments elsewhere in general if really appropriate.